autoload/maktaba/plugin.vim

```diff
@@ -950,8 +950,3 @@ endfunction
 function! maktaba#plugin#GetExtensionRegistry() dict abort
   return maktaba#extension#GetInternalRegistry(self.name)
 endfunction
-
-
-
-" Must be at the end of the file, to avoid infinite loops.
-let s:maktaba = maktaba#Maktaba()
```

Comment:
Should leave a note here or towards the top of the file like `NOTE: Do not call maktaba#Maktaba() here or it may cause circular imports. See https://github.com/google/vim-maktaba/issues/239`.

Reply:
The issue is addressed in a more general way now.